Lib/test/test_inspect.py

```diff
@@ -384,6 +384,12 @@ def test_cleandoc(self):
     def test_getcomments(self):
         self.assertEqual(inspect.getcomments(mod), '# line 1\n')
         self.assertEqual(inspect.getcomments(mod.StupidGit), '# line 20\n')
+        # If the object source file is not available, return None
+        fn = "_non_existing_filename_used_for_sourcefile_test.py"
```

Comment:
`_non_existing_filename.py` is both short and descriptive. Also, this way you can avoid using a `fn` variable.